torchci/lib/fetchCommit.ts

```diff
@@ -1,23 +1,22 @@
 import _ from "lodash";
 import { Octokit } from "octokit";
 import rocksetVersions from "rockset/prodVersions.json";
+import { queryClickhouseSaved, useClickhouse } from "./clickhouse";
 import { commitDataFromResponse, getOctokit } from "./github";
 import { removeCancelledJobAfterRetry } from "./jobUtils";
 import getRocksetClient from "./rockset";
 import { CommitData, JobData } from "./types";
 
-export default async function fetchCommit(
-  owner: string,
-  repo: string,
-  sha: string
-): Promise<{ commit: CommitData; jobs: JobData[] }> {
-  // Retrieve commit data from GitHub
-  const octokit = await getOctokit(owner, repo);
-  const rocksetClient = getRocksetClient();
-
-  const [githubResponse, commitJobsQuery] = await Promise.all([
-    octokit.rest.repos.getCommit({ owner, repo, ref: sha }),
-    await rocksetClient.queryLambdas.executeQueryLambda(
+async function fetchDatabaseInfo(owner: string, repo: string, sha: string) {
```

Comment:
This is a good feature IMO, let add this to the doc so that folks know that `useClickhouse` can be used to safely toggle ClickHouse.

(It seems that there is no easy way to make this generic for all queries, so each would probably need to implement its own `fetchDatabaseInfo` if needed)

Reply:
Looking at this, I think we should have this toggle on HUD itself so that we can quickly compare Rockset and ClickHouse.  This should be in a separated PR though I think